packages/stripe-graphql-js/src/schema/stripe.ts

```diff
@@ -64,7 +64,7 @@ builder.objectType('Stripe', {
           starting_after: startingAfter || undefined
         })
 
-        const customerData = customers.data.filter((customer) => isAllowed(customer.id, context))
+        const customerData = customers.data.filter(async (customer) => await isAllowed(customer.id, context))
```

Comment:
Unfortunately this won't work as `filter` doesn't support async functions :(
We could solve this with a reduced and `Promise.all`, but for the sake of readability, we could do something like this:

```suggestion
        const customerData: Stripe.Customer[] = []
        for (const customer of customers.data) {
          if (await isAllowed(customer.id, context)) {
            customerData.push(customer)
          }
        }
```

Reply:
My bad. Thanks for the suggestion.
Lets go for it :)